test/lit/wat-kitchen-sink.wast

```diff
@@ -2957,6 +2957,171 @@
+ ;; CHECK-NEXT:   (i32.const 1)
+ ;; CHECK-NEXT:   (ref.func $table-set)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (table.set $table-any
+ ;; CHECK-NEXT:   (i32.const 2)
+ ;; CHECK-NEXT:   (ref.null none)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $table-set
+  i32.const 0
+  ref.null func
+  table.set
+  i32.const 1
+  ref.func $table-set
+  table.set 1
+  i32.const 2
+  ref.null any
+  table.set $table-any
+ )
+
+ ;; CHECK:      (func $table-size (type $void)
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (table.size $timport$0)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (table.size $funcs)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (table.size $table-any)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $table-size
+  table.size
+  drop
+  table.size 1
+  drop
+  table.size $table-any
+  drop
+ )
+
+ ;; CHECK:      (func $table-grow (type $void)
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (table.grow $timport$0
+ ;; CHECK-NEXT:    (ref.null nofunc)
+ ;; CHECK-NEXT:    (i32.const 0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (table.grow $funcs
+ ;; CHECK-NEXT:    (ref.func $table-grow)
+ ;; CHECK-NEXT:    (i32.const 1)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (table.grow $table-any
+ ;; CHECK-NEXT:    (ref.null none)
+ ;; CHECK-NEXT:    (i32.const 2)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $table-grow
+  ref.null func
+  i32.const 0
+  table.grow
+  drop
+  ref.func $table-grow
+  i32.const 1
+  table.grow 1
+  drop
+  ref.null any
+  i32.const 2
+  table.grow $table-any
+  drop
+ )
+
+ ;; CHECK:      (func $table-fill (type $void)
+ ;; CHECK-NEXT:  (table.fill $timport$0
+ ;; CHECK-NEXT:   (i32.const 0)
+ ;; CHECK-NEXT:   (ref.null nofunc)
+ ;; CHECK-NEXT:   (i32.const 1)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (table.fill $funcs
+ ;; CHECK-NEXT:   (i32.const 2)
+ ;; CHECK-NEXT:   (ref.func $table-fill)
+ ;; CHECK-NEXT:   (i32.const 3)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (table.fill $table-any
+ ;; CHECK-NEXT:   (i32.const 4)
+ ;; CHECK-NEXT:   (ref.null none)
+ ;; CHECK-NEXT:   (i32.const 5)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $table-fill
+  i32.const 0
+  ref.null func
+  i32.const 1
+  table.fill
+  i32.const 2
+  ref.func $table-fill
+  i32.const 3
+  table.fill 1
+  i32.const 4
+  ref.null any
+  i32.const 5
+  table.fill $table-any
+ )
+
+ ;; CHECK:      (func $table-copy (type $void)
+ ;; CHECK-NEXT:  (table.copy $timport$0 $timport$0
+ ;; CHECK-NEXT:   (i32.const 0)
+ ;; CHECK-NEXT:   (i32.const 1)
+ ;; CHECK-NEXT:   (i32.const 2)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (table.copy $funcs $funcs
+ ;; CHECK-NEXT:   (i32.const 3)
+ ;; CHECK-NEXT:   (i32.const 4)
+ ;; CHECK-NEXT:   (i32.const 5)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $table-copy
+  i32.const 0
+  i32.const 1
+  i32.const 2
+  table.copy
+  i32.const 3
+  i32.const 4
+  i32.const 5
+  table.copy 1 $funcs
```

Comment:
Is it possible for just one of the tables to be specified?

Reply:
No, it's both or neither.